src/Silex/Provider/RoutingServiceProvider.php

```diff
@@ -23,6 +23,9 @@
 use Symfony\Component\Routing\RouteCollection;
 
 /**
+ * @deprecated Use `\Spryker\Zed\Router\Communication\Plugin\Application\RouterApplicationPlugin` instead.
```

Comment:
We shouldnt use leading \ on text-internal FQCN, those only apply to doc block values directly as per specs. Otherwise, they are never needed and discouraged.

Reply:
This must be done by a tool which will come... I personally use PHPStorm to get the reference to things like this and it is already annoying that I have to manually add the braces for method references when I need to also remove now a trailing backslash it's not getting better.

I don't see the reason why it is so bad to have it but on the other hand, I also do not care if there is one or not, I just choose the easiest way for me and probably others do so as well.

We can automate this and then we do not need to talk about those anymore :)

Reply:
Sure, we can automate this easily in the future. Until then we just have to check it manually for now.